api-testing/multicloud-api/src/main/java/com/soda/utils/BinaryUtils.java

```diff
@@ -0,0 +1,223 @@
+/*
+  Copyright 2020 The SODA Authors.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+ */
+package com.soda.utils;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.util.Locale;
+
+/**
+ * Utilities for encoding and decoding binary data to and from different forms.
+ */
+public class BinaryUtils {
```

Comment:
Where are we referencing/using these codes?

Reply:
This code will be use during signature creation.

Reply:
My question is If this code is currently in-use for this PR.
If yes, please give the reference code usage.
I suggest don't push code in advance. Please push code whenever it is being used.

Reply:
I have pushed this code as a basic automation framework code. This will be use in next PR.